docs/ingredients/accounts.md

```diff
@@ -0,0 +1,41 @@
+
+When creating transactions, accounts should be annotated as read-write or read-only accounts. Only set an account to 'writable (read-write)' if thet ransaction will make changes to the account. Marking an account as read-only enables parallel account processing between transactions. The Solana runtime permits read-only accounts to be read concurrently by multiple programs. If a program attempts to modify a read-only account, the transaction is rejected by the runtime. All Accounts to be written to or read must be passed to the program inside a transaction. If an on-chain program modifies a read-only account, the transaction will be reverted. The first account will always be read-write since it is used to cover transaction fees. This same account could be a wallet or keypair, and commongly referred to as the 'fee payer'. This fee payer would also be a signer on their account. More on signers coming up. Speaking of fees, accounts are charged rent. Rent?
+
+Lets start by saying that many types of accounts are required to post a minimum rent balance, thus exempting them from paying rent. This includes executable accounts such as programs and other types of account such as tokens. Basically there is a cost to storing the data, and because your placing some collateral up, rent will not get deducted. If and when these accounts are closed they can also collect any remaining balance left over. In general, Rent fees are charged for storing data on-chain. If an account is not rent exempt then a rent fee is deducted based on their space requirements. If an account has no balance left, it will be purged from memory by the runtime after the transaction. Fees are charged based on optimized calculations taking many variables into account, like space used for your account data (more rent and fee links to details below). For many circumstances you will want to create a rent-exempt account. Any account can be made rent-exempt if its balance is higher than some threshold that depends on the space it’s consuming. This can be calculated by calling a get minimum rent function. In most cases, you can close an account and send the remaining balance to a destination address. When sending transactions, some instructions like 'close' may require an account signer.
```

Comment:
We may want to just have a different ingredient for Rent and mention some of this information.

Reply:
rent only applies to account and is a integral part of accounts, so it was included here. it has to do with account setup, cost, and fees related to account specifically, and is a core piece of an account to know about.  we could make it its own section, but it does not really stand by itself, and is not used differently like other concepts are. Additionally it ties in closing accounts and what happens to the balance that may have been posted, tying things together. i think including a paragraph here and linking to any additional rent details is fine.